cloudant-spark-sql/build.sbt

```diff
@@ -37,3 +37,25 @@ assemblyMergeStrategy in assembly := {
 }
 
 assemblyJarName in assembly := "cloudant-spark.jar"
+
+
+spAppendScalaVersion := true
+
+spName := "com.cloudant.spark/cloudant-spark"
+
+sparkVersion  := "1.5.1"
+
+sparkComponents := Seq("sql")
+
+spShortDescription := "Spark SQL Cloudant External Datasource"
+
+spDescription := """Spark SQL Cloudant External Datasource.
+                   |Cloudant integration with Spark as Spark SQL external datasource.
+                   | - Allows to load data from Cloudant dabases and indexes into Spark.
+                   | - Allows to save resulting data from Spark to existing Cloudant databases.
+                   | - Supports predicates push down (only based on _id field in databases, but varios fields for indexes).
+                   | - Support column pruning for indexes.""".stripMargin
+
+credentials += Credentials(Path.userHome / ".ivy2" / ".sbtcredentials")
+
+
```

Comment:
I tried `sbt spPublish` but get an exception: java.lang.AssertionError: assertion failed: Please provide a license with the key licenses in your build.sbt file

Can we add `licenses += "Apache-2.0" -> url("http://opensource.org/licenses/Apache-2.0")` please?